Исправить сборку webpack

При запуске команды npm start вылетает такая ошибка

```Cannot read property 'eslintEnabled' of undefined
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! clue@0.1.0 start: node scripts/start.js
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the clue@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/killreal/.npm/_logs/2020-03-06T14_39_21_514Z-debug.log

--- a/scripts/start.js
+++ b/scripts/start.js
@@ -18,6 +18,7 @@ require('../config/env');
 const fs = require('fs');
 const chalk = require('react-dev-utils/chalk');
 const webpack = require('webpack');
+const minimist = require('minimist');
 const WebpackDevServer = require('webpack-dev-server');
 const clearConsole = require('react-dev-utils/clearConsole');
 const checkRequiredFiles = require('react-dev-utils/checkRequiredFiles');
@@ -43,6 +44,12 @@ if (!checkRequiredFiles([paths.appHtml, paths.appIndexJs])) {
 // Tools like Cloud9 rely on this.
 const DEFAULT_PORT = parseInt(process.env.PORT, 10) || 3000;
 const HOST = process.env.HOST || '0.0.0.0';
+
+const options = minimist(process.argv.slice(2));
+
+const processAgruments = ({eslintEnabled}) => ({
+  eslintEnabled: eslintEnabled ? eslintEnabled.toLowerCase() === 'true' : true,
+});
 
 if (process.env.HOST) {
   console.log(
@@ -76,7 +83,7 @@ checkBrowsers(paths.appPath, isInteractive)
       return;
     }
 
-    const config = configFactory('development');
+    const config = configFactory('development', processAgruments(options));
     const protocol = process.env.HTTPS === 'true' ? 'https' : 'http';
     const appName = require(paths.appPackageJson).name;
     const useTypeScript = fs.existsSync(paths.appTsConfig);

